health-services/project-factory/src/server/api/campaignApis.ts

```
@@ -498,13 +537,13 @@
         });
       }
-      count++;
+      mobileNumbercount++;
     }
   }
-  if (count) {
+  if (mobileNumbercount || userNameCount) {
     request.body.ResourceDetails.status = "invalid";
   }
-  logger.info("Invalid resources count : " + count);
+  logger.info("Invalid resources count : " + mobileNumbercount + userNameCount);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)